pkg/scheduler/controller/shoot/scheduler.go

```diff
@@ -70,21 +70,11 @@ func NewGardenerScheduler(
 		return nil, fmt.Errorf("failed to get Shoot Informer: %w", err)
 	}
 
-	var (
-		coreV1beta1Informer = gardenCoreInformerFactory.Core().V1beta1()
-
-		seedInformer         = coreV1beta1Informer.Seeds()
-		seedLister           = seedInformer.Lister()
-		cloudProfileInformer = coreV1beta1Informer.CloudProfiles()
-		cloudProfileLister   = cloudProfileInformer.Lister()
-		shootQueue           = workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(config.Schedulers.Shoot.RetrySyncPeriod.Duration, 12*time.Hour), "gardener-shoot-scheduler")
-	)
-
 	schedulerController := &SchedulerController{
 		k8sGardenCoreInformers: gardenCoreInformerFactory,
```

Comment:
Can be removed now, I guess.

Reply:
Nice catch, done! Please check d8b3f17 and b76327c.